### ISSUE TITLE
Using AES256, how can I get iv in my code?

### ISSUE THREAD

**Issue by Dimius555** [leif-ibsen/SwiftECC#25]
I'm using the next code:

```
func test() {
            let domain = Domain.instance(curve: .EC256k1)
            let (pubKey, privKey) = domain.makeKeyPair()
            let pubPEM = pubKey.pem
            let privPEM = privKey.pem

            print(pubPEM)
            print(privPEM)
 
            let object = MyObject()
            let jsonEncoder = JSONEncoder()
            let jsonData = try! jsonEncoder.encode(object)
            let json = String(data: jsonData, encoding: String.Encoding.utf8)

            let message = json!.data(using: .utf8)!

            let sig = privKey.sign(msg: message)
            let ok = pubKey.verify(signature: sig, msg: message)
            print(sig)

            let encrypted = pubKey.encrypt(msg: message, cipher: .AES256)
            print(encrypted)

            let decrypted = try! privKey.decrypt(msg: encrypted, cipher: .AES256)
            print(String(data: decrypted, encoding: .utf8))
}
```

How can I get iv here? It's required by the server

**Comment by leif-ibsen** [comment]
When you encrypt something using ECIES and GCM, the IV is computed in a random way
and its value is included in the encrypted data so that it is possible to decrypt it again.
The API does not give you access to the actual IV value.

**Comment by leif-ibsen** [comment]
This is fixed in release 3.4.0 where the new method 'getKeyAndMac' in the ECPrivateKey class
retrieves the AES key and HMAC key (IV)
